lib/wallaby/driver.ex

```diff
@@ -223,10 +256,11 @@ defmodule Wallaby.Driver do
     %{using: "css selector", value: css_selector}
   end
 
-  defp request(method, url, params \\ %{}) do
+  defp request(method, url, params \\ %{}, opts \\ []) do
     headers = [{"Content-Type", "text/json"}]
     body = case params do
       params when map_size(params) == 0 -> ""
+      params when [{:encode_json, false}] == opts -> params
```

Comment:
is there a way to match a keyword list that might also include other values inside the guard? that's not possible right? you'd need another case statement or to pattern match on the function signature?